Sources/Datadog/Datadog.swift

```diff
@@ -166,49 +224,113 @@ public class Datadog {
         core?.clearAllData()
     }
 
+    /// Initializes the Datadog SDK.
+    ///
+    /// - Parameters:
+    ///   - configuration: the SDK configuration.
+    ///   - trackingConsent: the initial state of the Data Tracking Consent given by the user of the app.
+    ///   - instanceName: The core instance name.
+    public static func initialize(
+        with configuration: Configuration,
+        trackingConsent: TrackingConsent,
+        instanceName: String = CoreRegistry.defaultInstanceName
+    ) {
+        // TODO: RUMM-511 remove this warning
+        #if targetEnvironment(macCatalyst)
+        consolePrint("⚠️ Catalyst is not officially supported by Datadog SDK: some features may NOT be functional!")
+        #endif
+
+        do {
+            try initializeOrThrow(
+                with: configuration,
+                trackingConsent: trackingConsent,
+                instanceName: instanceName
+            )
+        } catch {
+            consolePrint("\(error)")
+        }
+    }
+
     private static func initializeOrThrow(
-        initialTrackingConsent: TrackingConsent,
-        configuration: FeaturesConfiguration,
+        with configuration: Configuration,
+        trackingConsent: TrackingConsent,
         instanceName: String
     ) throws {
-        if Datadog.isInitialized {
+        if CoreRegistry.default is DatadogCore {
             throw ProgrammerError(description: "SDK is already initialized.")
         }
 
-        let serverDateProvider = configuration.common.serverDateProvider ?? DatadogNTPDateProvider()
+        let debug = configuration.processInfo.arguments.contains(LaunchArguments.Debug)
+        if debug {
+            consolePrint("⚠️ Overriding verbosity, and upload frequency due to \(LaunchArguments.Debug) launch argument")
+            Datadog.verbosityLevel = .debug
```

Comment:
question/ @maxep What is the plan for getting rid of global `.verbosityLevel`? Do we have necessary items in the backlog?

Reply:
Is there a plan to get rid of it? 🤔 
Is is used by `DD.logger` which is also global and since it's for development it makes sense keep it this way, no?

Reply:
TBC: sorry, I don't mean "removing it" 🙂. Do we consider attaching `.verbosityLevel` to SDK core instance, or we want to keep it global? Global would mean seeing logs from all instances of the SDK (even not mine). Whatever we decide it is rather low priority.

Reply:
Good idea 👍